packages/scan/src/requests/UploadFile.ts

```diff
@@ -0,0 +1,23 @@
+import FormData from 'form-data';
+import { HttpOptions, HttpRequest } from '@secbox/bus';
+
+export interface UploadFileOptions {
+  filename: string;
+  content: string;
+}
+
+export class UploadFile extends HttpRequest<FormData, { id: string }> {
+  constructor(fileOptions: UploadFileOptions, query: Record<string, unknown>) {
```

Comment:
The only param that might be passed in `query` is `discard`. You don't need to pass a whole hashmap with a certain number of additional params.
```suggestion
  constructor(options: UploadHarPayload) {
```
where `UploadHarPayload` is

```ts
export interface UploadHarPayload {
  har: Har;
  filename: string;
  discard?: boolean;
}
```